app/routers/profile.py

```diff
@@ -138,6 +138,23 @@ async def update_telegram_id(
     return RedirectResponse(url=url, status_code=HTTP_302_FOUND)
 
 
+@router.post("/privacy")
+async def update_calendar_privacy(
+        request: Request,
+        session=Depends(get_db)
+):
+    user = session.query(User).filter_by(id=1).first()
+    data = await request.form()
+    new_privacy = data['privacy']
+
+    # Update database
```

Comment:
you can delete the comment

Reply:
This comment appears in all the other forms so I just kept the same structure :) Unless a decision is made to remove the comments from all the file I think this should stay